src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.commons.util;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+/**
+ * A class for date time manipulation.
+ */
+public class DateUtil {
+    /**
+     * Converts to the local date time format of the month and year combined.
```

Comment:
I would think `Converts to the local date time format from supplied {@code Month} and {@code Year}`.

Reply:
Changed to `Gets the first day of the {@code Month} and {@code Year} in local date time format.` instead due to the change in name.